examples/npm-stats/README.md

```diff
@@ -1,16 +1,48 @@
-# KPI tracker
+# npm stats
 
-<p class="description">This Analytics dashboard shows how to track a KPI from a third party data source.</p>
+<p class="description">This analytics dashboard shows how to track a KPI from a third-party data source.</p>
 
-{{"component": "modules/components/DocsImage.tsx", "src": "/static/toolpad/marketing/npm-stats.png", "alt": "Analytics dashboard", "caption": "Analytics dashboard showing download stats of Toolpad NPM package"}}
+<a href="https://npm-stats.onrender.com/prod/pages/evZC-gp" target="_blank">
+  <img src="https://mui.com/static/toolpad/docs/examples/npm-stats.png" alt="npm stats app" style="aspect-ratio: 1.8" width="100%">
+</a>
+
+## Check out the live app
```

Comment:
The content is confusing, the first link is the editor, the second is the live app, two different things.

The order feels a bit backward. I think that in the best-case scenario, people run the project in their local environment, not in StackBlitz. First because this is how the product is mean to be used, locally, not in StackBlitz, and second because it's broken in StackBlitz.